openapi/event-reporting/api-spec.json

```diff
@@ -353,13 +353,13 @@
             "example": "2023-01-01T13:00:00Z",
             "default": "null"
           },
-          "leftStopPlaceRef": {
+          "leftCancelledStopPlaceRef": {
             "type": "string",
             "description": "Required if `category` is `CANCELLATION` and `subcategory` is `PARTIAL`, ignored\notherwise. An NSR reference to one of the 2 stop places delimiting the sequence of\nstops affected by the partial cancellation, or to the single stop if only one stop is\naffected. See `rightStopPlaceRef`.\n",
             "example": "NSR:StopPlace:123",
             "default": "null"
           },
-          "rightStopPlaceRef": {
+          "rightCancelledStopPlaceRef": {
             "type": "string",
             "description": "Required if `category` is `CANCELLATION` and `subcategory` is `PARTIAL`, ignored\notherwise. An NSR reference to the other of the 2 stop places delimiting the sequence\nof stops affected by the partial cancellation, or to the single stop if only one stop\nis affected. See `leftStopPlaceRef`.\n",
```

Comment:
Please update the description for this field to reference the new name:

`leftCancelledStopPlaceRef`

Reply:
Fixed!